pdcupdater/handlers/retirement.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
@bowlofeggs, could you open an issue about this? It's a good observation, but not something I want to handle in this PR.